docs/assets/main.scss

```diff
@@ -16,7 +16,6 @@ code {
   padding-left: 0.25em;
   font-family: 'Source Code Pro', monospace;
   font-weight: 400;
-  white-space: nowrap;
```

Comment:
I might be imagining this, but I feel as though I've seen this change in a few PRs lately. I am not sure if/how _you_ would know this, but.... does that seem familiar to you? do you know what's going on with this change?

Reply:
Here's details in the commit message :) https://github.com/learningequality/kolibri-design-system/pull/748/commits/f0a1dad8dcd4d404e0f6816f71ad091df3178451

Visually, this change allows preserving nice format for the `layoutConfig` example here
![Screenshot from 2024-09-13 17-47-19](https://github.com/user-attachments/assets/9e121ee8-64ab-40e7-a098-ab4ffd44c3c7)

I don't know why it was there originally though, but haven't noticed any issues yet after removing.

Reply:
If I recall, it would show on one line otherwise, which is not readable (we can still achieve one line format by formatting the source docstring accordingly - so this is about more precise transfer of the source string to auto-generated docs)